File: Sitio Privado/app/domain/SiteInformation.ts
module app.domain {
    export interface ISiteInformation {
        id: number;
        userId: number;
        abbreviateName: string;
        siteName: string;
        siteType: string;
        description: string;
        url: string;
        cn: string;
        priority: number;
    }

    export class SiteInformation extends app.domain.EntityBase implements ISiteInformation {
        constructor(public id: number,
            public userId: number,
            public abbreviateName: string,
            public siteName: string,
            public siteType: string,
            public description: string,
            public url: string,
            public cn: string,
            public priority: number) {

            super();
        }
    }
}

File: Sitio Privado/Models/SiteInformation.cs
namespace Sitio_Privado.Models
{
    public class SiteInformation
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string AbbreviateName { get; set; }
        public string SiteName { get; set; }
        public string SiteType { get; set; }
        public string Description { get; set; }
        public string Url { get; set; }
        public string Cn { get; set; }
        public int Priority { get; set; }
    }
}